src/rds_scheduler.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'bundler'
+Bundler.setup
+
+require 'aws-sdk-rds'
+require 'logger'
+
+require_relative 'rds_helper'
+require_relative 'time_parser'
+
+# Manage uptime schedules for RDS Instances
+class RDSScheduler
+  attr_reader :loop_interval,
```

Comment:
Doesn't look like you need to expose any of these outside of the class? Can just refer to the internal `@xxxx` vars internally?

Reply:
Ah yes good spot, will update.